### PR TITLE
Releasing Realm Web 1.0.0-rc.1

### PR DESCRIPTION
**Pull request by kraenhansen**
## What, How & Why?

Realm Web v1.0.0-rc.1 was released from this branch, merging it to make it official 😄 

## ☑️ ToDos
* [x] 📝 Changelog entry
* [x] 📝 `Compatibility` label is updated or copied from previous entry
* [x] 🚦 Tests
* [x] 📝 Public documentation PR created or is not necessary
* [x] 💥 `Breaking` label has been applied or is not necessary

*If this PR adds or changes public API's:*
* [x] typescript definitions file is updated
* [x] jsdoc files updated
* [x] Chrome debug API is updated if API is available on React Native

### CODE REVIEW

File: .github/workflows/publish-realm-web.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Publish Realm Web
+
+on:
+  create:
+    tags:
+      - realm-web-v*
```
Comment (steffenagger) [inline .github/workflows/publish-realm-web.yml:6]: Our CI for realm-js currently picks up _any_ tag & build for release, right? Or am I jumping to conclusions here?
If not, we should probably streamline the naming of these tags.

**Review by steffenagger — APPROVED** [review]
LGTM
(though I think tagging for realm-web `realm-web-v1.0.0-rc.1` would currently trigger a CI build in `realm-js` for release - if not caught by the ./packages git-check)

File: packages/realm-web/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
 ?.?.? Release notes (2020-??-??)
 =============================================================
 
+### Breaking Changes
```
Comment (blagoev) [inline packages/realm-web/CHANGELOG.md:4]: are we releasing with these on top. Why not add them after the release ?

**Review by blagoev — COMMENTED** [review]
Should probably add the empty changelog template after the release

**Comment by kraenhansen** [comment]
> Should probably add the empty changelog template after the release

For now the changelog hasn't been automated, but we should definitely automate that eventually.

File: packages/realm-web/CHANGELOG.md (hunk 2 above)
Comment (kraenhansen) [reply packages/realm-web/CHANGELOG.md:4]: Right. I agree this should probably have been added on a later commit.

**Pull request merged**